checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java

```diff
@@ -1495,6 +1529,7 @@ private void incrementNumMustCall(Element elt) {
    *
    * @param type the type of the object that has a must call obligation
    */
+  // TODO: Should this be removed?  It doesn't seem like some thing a user needs.
```

Comment:
We've left it in as an undocumented feature on the theory that if we want to re-run the experiments from the RLC paper it will be useful. I have a slight preference for keeping it, but I'm also willing to remove it if you think maintaining it will be more effort than it's worth.

Reply:
It's fine to keep it.